src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -138,9 +191,28 @@ export default function withAnnotations<P extends object>(
             }
         };
 
-        handleAnnotator = (annotator: Annotator): void => {
+        handleAnnotationsFetchError = ({ error }: { error: Error }): void => {
+            if (!error) {
+                return;
+            }
+
+            this.setState({
```

Comment:
Does this need to be set in state or can we just immediately trigger `this.props.onError`?